frontend/client/src/screens/CodeValidations.js

```diff
@@ -70,6 +70,15 @@ const CodeValidationsBase = observer((props) => {
       setNeedsReset(true)
       updateButtonDisabled()
       countdown()
+
+      // WORK IN PROGRESS, notes:
+      // need to import analytics?  and make analytics testable in dev by removing conditional in firebase.js and resolve local error caused by removing it
```

Comment:
> need to import analytics?

We can attach it to the store as a property called `analytics`, and then access it anywhere in the app via `props.store.analytics`.

> and make analytics testable in dev by removing conditional in firebase.js and resolve local error caused by removing it

Yeah, just remove the conditional and see what happens on infrastructure where analytics isn't enabled.